tooling/components/io.siddhi.distribution.editor.core/src/main/resources/web/js/operator-finder/operator-finder.js

```diff
@@ -138,10 +212,10 @@ define(['jquery', 'lodash', 'log', 'handlebar', 'designViewUtils', 'app/source-e
             for (type in meta.inBuilt) {
                 if (meta.inBuilt.hasOwnProperty(type)) {
                     meta.inBuilt[type].forEach(function (operator) {
-                        var op = _.clone(operator);
-                        op.fqn = op.name;
-                        op.type = type;
-                        operators.push(op);
+                        var cloneOperator = _.clone(operator);
+                        cloneOperator.fqn = cloneOperator.name;
+                        cloneOperator.type = type;
+                        operators.push(mdConversion(cloneOperator));
```

Comment:
```suggestion
                        operators.push(createMDFile(cloneOperator));
```

Reply:
960249d this commit fix above request change